### ISSUE TITLE
prometheus exporter doesn't provide any HELP lines to explain the metrics

### ISSUE THREAD

**Issue by matthewdarwin** [AntelopeIO/leap#1063]
There should be a HELP section for each metric explaining what it means.  This is the current output.  I don't know what all these metrics mean actually.

```
$ curl http://localhost:9101/v1/prometheus/metrics
# TYPE exposer_transferred_bytes_total counter
exposer_transferred_bytes_total 8938
# TYPE exposer_scrapes_total counter
exposer_scrapes_total 12
# TYPE dropped_trxs counter
dropped_trxs 0
# TYPE blocks_produced counter
blocks_produced 0
# TYPE trxs_produced counter
trxs_produced 0
# TYPE num_calls gauge
num_calls 0
# TYPE num_peers gauge
num_peers 6
# TYPE num_clients gauge
num_clients 2
# TYPE unapplied_transactions gauge
unapplied_transactions 1
# TYPE blacklisted_transactions gauge
blacklisted_transactions 0
# TYPE last_irreversible gauge
last_irreversible 72589136
# TYPE head_block_num gauge
head_block_num 72589461
# TYPE subjective_bill_account_size gauge
subjective_bill_account_size 0
# TYPE scheduled_trxs gauge
scheduled_trxs 1
```

**Comment by matthewdarwin** [comment]
num_clients is what exactly?  Number of HTTP connections active I guess? Not exactly obvious.  The release notes don't really explain either... https://github.com/AntelopeIO/leap/releases/tag/v4.0.0-rc2

**Comment by heifner** [comment]
Included as part of #1074
